src/compiler/scala/tools/nsc/transform/Erasure.scala

```diff
@@ -422,7 +422,13 @@ abstract class Erasure extends AddInterfaces
     val opc = enteringExplicitOuter {
       new overridingPairs.Cursor(root) {
         override def parents              = List(root.info.firstParent)
-        override def exclude(sym: Symbol) = !sym.isMethod || super.exclude(sym)
+        // Varargs bridges may need generic bridges due to the non-repeated part of the signature of the involved methods.
+        // The vararg bridge is generated during refchecks (probably to simplify override checking),
+        // but then the resulting varargs "bridge" method is considered an artifact (why???),
+        // causing `super.exclude` to veto it from our bridge generation here, unless we explicitly include it (hence `!sym.hasFlag(VBRIDGE) &&`).
+        // TODO: could we generate just one bridge method that does both the java-vararg-array <-> scala-repeated-param-seq conversion
+        //       as well as the casts from the erased types of the overridden generic method (regular bridge method)?
+        override def exclude(sym: Symbol) = !sym.isMethod || (!sym.hasFlag(VBRIDGE) && super.exclude(sym))
```

Comment:
I think artifacts are intended to be skipped while type checking (for performance? because they may have incorrect types? because their types are trivially correct?)